pkgs/tools/security/bitwarden/default.nix

```
@@ -0,0 +1,55 @@
+{ stdenv, fetchurl, makeWrapper, electron_5, libsecret, dpkg, gtk3, glib, gsettings-desktop-schemas, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  name = "bitwarden";
+  version = "1.15.2";
+
+  src = fetchurl {
+    url = "https://github.com/bitwarden/desktop/releases/download/v${version}/Bitwarden-${version}-amd64.deb";
+    sha256 = "0yz4hkqqwq2zrdjfxk5kybhs90n80k6bkn0625m47b09lwl2di4f";
+  };
+
+  nativeBuildInputs = [
+    dpkg
+    makeWrapper
+    wrapGAppsHook
+    libsecret
+  ];
+
+  buildInputs = [
+    glib
+    gsettings-desktop-schemas
+    gtk3
+  ];
+
+  unpackPhase = "dpkg-deb -x $src .";
+
+  dontWrapGApps = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin $out/share
+    {
+    mv opt/Bitwarden $out/share/Bitwarden
+    mv usr/share/{applications,icons,doc} $out/share/
+    }
+
+    runHook postInstall
+  '';
+
+  postFixup = ''
+    makeWrapper ${electron_5}/bin/electron $out/bin/bitwarden \
+    --add-flags $out/share/Bitwarden/resources/app.asar \
+    "''${gappsWrapperArgs[@]}" \
+    --prefix LD_LIBRARY_PATH : "${stdenv.lib.makeLibraryPath [ stdenv.cc.cc libsecret ]}"
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A secure and free password manager for all of your devices.";
```

```suggestion
    description = "A secure and free password manager for all of your devices";
```

See contributing guidelines, no periods at the end of `description`.